src/components/Reactions/EmojiReactionBubble.tsx

```diff
@@ -82,7 +83,8 @@ function EmojiReactionBubble(
             accessible
             dataSet={{[CONST.SELECTION_SCRAPER_HIDDEN_ELEMENT]: true}}
         >
-            <Text style={[styles.emojiReactionBubbleText, StyleUtils.getEmojiReactionBubbleTextStyle(isContextMenu)]}>{emojiCodes.join('')}</Text>
+            {/* TODO: investigate why emojis are moved up on iOS on the new architecture */}
+            <Text style={[styles.emojiReactionBubbleText, StyleUtils.getEmojiReactionBubbleTextStyle(isContextMenu), {transform: [{translateY: Platform.OS === 'ios' ? 2 : 0}]}]}>{emojiCodes.join('')}</Text>
```

Comment:
Also, again using `Platform.OS` is prohibited

Reply:
Changed it to use a helper function instead